recipes-graphics/flutter-engine/flutter-engine_git.bb

```diff
@@ -209,10 +209,10 @@ do_configure() {
 
         # make it easy to parse
         BUILD_DIR="$(echo ${TMP_OUT_DIR} | sed "s/_RUNTIME_/${MODE}/g")"
-        ARGS_FILE="${WORKDIR}/src/${BUILD_DIR}/args.gn"
+        ARGS_FILE="${S}/${BUILD_DIR}/args.gn"
```

Comment:
leave as is, so PR is specific to changes required for `master`.  This is a style change